catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/SegmentHandler.java

```diff
@@ -34,6 +39,9 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SegmentHandler.class);
 
+  private static final Set<String> COUNTRY_CODE_ATTRIBUTES =
+      ImmutableSet.of(Security.RELEASABILITY, Security.OWNER_PRODUCER);
```

Comment:
❓ Should this contain `Location.COUNTRY_CODE` too or is it only for `Security` attributes?

Reply:
I think there's one NITF TRE that maps there, but it's expecting a FIPS country code and does the conversion in the parser. That kind of goes against what I'm doing here though which is to maintain the ext nitf attrs and only normalize the catalog taxonomy ones